root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1153,12 +1153,6 @@ const testData = [
                      input_url: 'http://www.dailymotion.com/playlist/xwvuts_who-knows_top/1#video=xyztuvw',
             expected_clean_url: 'https://www.dailymotion.com/video/xyztuvw',
   },
-  // Decoda
-  {
-                     input_url: 'http://decoda.com/robi-on-ne-meurt-plus-damour-lyrics',
-             input_entity_type: 'work',
-    expected_relationship_type: 'lyrics',
-  },
```

Comment:
Maybe we can do more and completely blacklist `decoda.com` URLs for any type of relationship.